tests/hw-accel/nfd/internal/wait/wait.go

```diff
@@ -7,7 +7,11 @@ import (
 
 	"github.com/openshift-kni/eco-goinfra/pkg/clients"
 	"github.com/openshift-kni/eco-goinfra/pkg/nodes"
-	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+
+	"github.com/golang/glog"
```

Comment:
You can take the `github.com/golang/glog` and get it out from the group of `openshift-kni`. It will look more pretty